lib/cache-manager.js

```diff
@@ -0,0 +1,57 @@
+const debug = require('debug')('jambonz:db-helpers');
+
+const cache = {};
+let timerID = null;
+
+function startUpdater() {
+  // debug('cache: maybe update', { timerID, envVar: process.env.JAMBONES_MYSQL_REFRESH_TTL });
+
+  if (timerID === null) {
+    debug('cache: updating');
+    const ttl = parseInt(process.env.JAMBONES_MYSQL_REFRESH_TTL || '0.001', 10) * 1000;
```

Comment:
If we actually reach this code, isn't it the case that JAMBONES_MYSQL_REFRESH_TTL is defined (and is non-zero)?  I am unclear what the purpose of the default is here